.release-please-manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  ".": "1.18.0",
-  "packages/api-explorer": "0.9.23",
-  "packages/code-editor": "0.1.14",
+  ".": "1.19.0",
+  "packages/api-explorer": "0.9.24",
+  "packages/code-editor": "0.1.15",
   "packages/extension-api-explorer": "21.20.0",
   "packages/extension-sdk": "21.20.0",
   "packages/extension-sdk-react": "21.20.0",
-  "packages/extension-utils": "0.1.0",
+  "packages/extension-utils": "21.20.0",
```

Comment:
this version is not correct

Reply:
I'm not clear. What should the version be.

Reply:
I'm thinking it should be `0.2.0` since there was a feature added. But when the major version is zero we can change that.

Reply:
We have set `"bump-patch-for-minor-pre-major": true` so it should be `0.1.1`.